xclim/indicators/atmos/_temperature.py

```diff
@@ -467,6 +470,7 @@ class TempWithIndexing(ResamplingIndicatorWithIndexing):
     "consecutive days with mean daily temperature below a given threshold.",
     cell_methods="",
     compute=indices.cold_spell_days,
+    parameters={"op": {"default": "<"}},
```

Comment:
I haven't checked them all, but you don't need to pass a "parameter" arg here if you are not changing anything. For  `cold_spell_days`, the default already is "<", so we usually do not repeat the information here.

Reply:
If `parameters` is not passed to the indicator wrapper, is it still possible for a user to change/control these values? I figured why not offer this as a possible option, but if it's too much in your opinion, I don't mind reverting some of these changes.

Reply:
When `parameters` are passed as dict, it simply overrides (updates) the configuration parsed from the indice's docstring and signature. So yes, even if it is not passed as you did, the argument still appears in the indicator's signature, and it is modifiable.

That provision is meant mostly to be used when we want to _inject_ a parameter and remove it from the indicator's signature. For example, in the same file you can see `daily_freezethaw_cycles`, which is a stricter version of `multiday_temperature_swing` : 
```
daily_freezethaw_cycles = TempWithIndexing(
   [...]
    parameters={
        "op": "sum",
        "window": 1,
        "thresh_tasmax": {"default": "0 degC"},
        "thresh_tasmin": {"default": "0 degC"},
        "op_tasmax": {"default": ">"},
        "op_tasmin": {"default": "<="},
    },
)
```
Here the `op` and `window` parameters are no longer controllable, but the others have only seen their default value change.

Reply:
I see! This demystifies the behaviour. Thanks for the explanation!